### PR TITLE
docs: add ESM usage example in Getting Started

### PR DESCRIPTION
**Pull request by atilagulers**
This commit updates the 'Getting Started' guide to include an ESM usage example, specifically demonstrating the export of routes using ESM syntax. A code snippet showing the addition of "type": "module" in a configuration example is also provided to assist users in setting up their projects for ESM.

This addresses the confusion mentioned in issue #5292 regarding ESM configuration in Fastify projects.

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
the markdown is not passing linting

File: docs/Guides/Getting-Started.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ fastify.listen({ port: 3000 }, function (err, address) {
 })
 ```
 
+>If you're using ECMAScript Modules (ESM) in your project, ensure to
```
Comment (Fdawgs) [inline docs/Guides/Getting-Started.md:58]:
```suggestion
> If you are using ECMAScript Modules (ESM) in your project, be sure to
```

**Review by Fdawgs — CHANGES_REQUESTED** [review]

**Comment by atilagulers** [comment]
@Fdawgs  i've applied changes that you suggested.

**Review by Fdawgs — APPROVED** [review]

**Review by metcoder95 — APPROVED** [review]

**Pull request merged**